doc/crds.md

```diff
@@ -614,6 +614,29 @@ Notable attributes:
 * **status.state**: Either of `PENDING`, `READY` or `ERROR`. If the state is `ERROR`, the
   attribute `status.errorMessage` contains the reason for the failure.
 
+While it's possible to extend a profile and build it based on another one, it's also
+possible to write a profile from scratch using the `TailoredProfile` construct.
+To do this, remember to set an appropriate title and description. It very important
```

Comment:
typo: `It's very important`

It might be good to put some example TailoredProfile yaml snippets to clarify what is expected.

Reply:
I plan to actually amend the CRDs (in a subsequent PR) to disallow empty titles and description. The XCCDF standard allows it, but it errors out in OpenSCAP. So let's just validate it on our CRDs